CHANGES.md

```diff
@@ -1,9 +1,20 @@
-=======================
```

Comment:
Don't think we should change the CHANGES file from restructuredText to markdown. All our changelog files are rst.
Also, this change is unrelated to the PR :)